ci-operator/config/openshift/release/openshift-release-master__ocp-4.6.yaml

```diff
@@ -16,6 +16,10 @@ base_images:
     namespace: ocp
     tag: upi-installer
 releases:
+  initial:
```

Comment:
This feels overly generic to me for "previous minor".  Can we do something that will leave space for 4.6-stable->4.6-nightly-candidate periodics as well?  Maybe `previousMinorStable`?  And then... however we tie that in as the initial release for the `e2e-vsphere-upgrade` job.  With `OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: "release:previousMinorStable"` or some such...

Reply:
@wking you tell me what you want and I will add it ;-)

Minimally I would like `4.next-1.x` to `4.next.x`.